fms/modules/attention.py

```diff
@@ -3,7 +3,7 @@
 
 import torch
 import torch.distributed
-from float8_experimental import Float8Tensor
+from float8_experimental.float8_tensor import hp_tensor_and_scale_to_float8
```

Comment:
it looks like the project has moved and should now be `torchao.float8` ?

we probably also need to add to requirements.txt

Reply:
looks like it.. let me see if it's just a quick renaming required